disciplines/isq.ttl

```diff
@@ -90,6 +108,30 @@ email: emanuele.ghedini@unibo.it"""@en ,
                                            skos:prefLabel "Velocity"@en .
 
 
+###  http://emmo.info/emmo#EMMO_034bc7dd_a8c2_4ed0_8b51_66ac9b00342f
+:EMMO_034bc7dd_a8c2_4ed0_8b51_66ac9b00342f rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_c29603f5_95e8_42f5_ab0c_f3bcf3166d53 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_3227b821_26a5_4c7c_9c01_5c24483e0bd0
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "12-4" ;
+                                           :EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "angle between the scattered ray and the lattice plane"@en ;
+                                           skos:prefLabel "BraggAngle"@en .
+
+
+###  http://emmo.info/emmo#EMMO_03d32783_3780_43c3_8b73_7efe9aeeed54
+:EMMO_03d32783_3780_43c3_8b73_7efe9aeeed54 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_c29603f5_95e8_42f5_ab0c_f3bcf3166d53 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_b3600e73_3e05_479d_9714_c041c3acf5cc
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "12-1.2" ;
+                                           :EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "fundamental translation vectors for the crystal lattice"@en ;
+                                           skos:prefLabel "FundamentalLatticeVectors"@en .
```

Comment:
Preferred labels should be singular noun, so a better prefLabel would be "FundamentalLatticeVector"

Reply:
done